frontend/src/components/details/CompareVariablesComponent.jsx

```diff
@@ -15,72 +16,18 @@ const CompareVariablesComponent = (props) => {
                 <h4>Select variables to compare:</h4>
             </div>
             <div className='w3-col l3 m3 s3 w3-left'>
-                <div className='w3-dropdown-hover '>
-                    <button
-                        className='w3-button w3-2018-sailor-blue'
-                        style={{
-                            fontSize: theme.typography.buttontitle.fontSize,
-                            fontWeight: theme.typography.buttontitle.fontWeight,
-                            lineHeight: theme.typography.buttontitle.lineHeight,
-                            letterSpacing: theme.typography.buttonSendtitle.letterSpacing
-                        }}
-                    >
-                        {state.selectedVariable}
-                    </button>
-                    <div
-                        className='w3-dropdown-content w3-bar-block w3-border'
-                        style={{
-                            fontSize: theme.typography.buttontitle.fontSize,
-                            fontWeight: theme.typography.buttontitle.fontWeight,
-                            lineHeight: theme.typography.buttontitle.lineHeight,
-                            letterSpacing: theme.typography.buttonSendtitle.letterSpacing
-                        }}
-                    >
-                        {state.variables.map((item) => (
-                            <button
-                                onClick={() => props.firstDropDownHandler(item.name)}
-                                className='w3-bar-item w3-button'
-                                key={item.id}
-                            >
-                                {item.name}
-                            </button>
-                        ))}
-                    </div>
-                </div>
+                <DropDownComponent
+                    selectedVariable={state.selectedVariable}
+                    variables={state.variables}
+                    dropDownHandler={props.firstDropDownHandler}
+                />
```

Comment:
Where is this Component being used in our Webpage?

Reply:
nowhere!!!!! 
this change was for when this branch's base was for the last sprint.

Reply:
let's don't delete this beautiful component.